pkg/lib/machines.js

```diff
@@ -593,6 +620,7 @@
                         open = true;
                         if (url)
```

Comment:
We can get the checksum also from the /packages API.  Do we need to see the ETag for other reasons?

The main reason for still using HTTP GET to get the initial manifests is that not all bridges support the /packages API.  Do you think we should try D-Bus first and only fall back to HTTP GET when /packages is not supported?